src/Proto3/Suite/DotProto/Internal.hs

```diff
@@ -61,22 +75,37 @@ dieLines (Turtle.textToLines -> msg) = do
 -- >>> toModulePath "foo/bar/././baz/../boggle.proto"
 -- Right (Path ["Foo","Bar","Boggle"])
 --
+-- >>> toModulePath "./foo.proto"
+-- Right (Path ["Foo"])
+--
+-- NB: We ignore preceding single '.' characters
+-- >>> toModulePath ".foo.proto"
+-- Right (Path ["Foo"])
 toModulePath :: FilePath -> Either String Path
-toModulePath fp
+toModulePath fp0@(fromMaybe fp0 . FP.stripPrefix "./" -> fp)
```

Comment:
This is slick. I didn't realize the binding from `@` would be in scope for a view pattern though it makes sense when I think about it (I guess it expands that to an openly recursive let binding?)